src/main/java/seedu/address/model/task/Type.java

```diff
@@ -30,10 +43,37 @@ public Type(String type) {
     }
 
     /**
-     * Returns true if a given string is a valid email.
+     * Returns true if a given string is a valid type.
+     */
+    public static boolean isValidType(String testType) {
+        if (testType == null) {
```

Comment:
iirc there's a "requireNonNull" method there? Can we use it?

Reply:
excellent suggestion, it works as well and is more concise